apps/rings-next/components/ring/RingCard.tsx

```diff
@@ -54,23 +57,47 @@ const RingCard: React.FC<RingCardProps> = ({ ring, id, editable = true, ...other
         const { stone1: stone1Changed, stone2: stone2Changed, stone3: stone3Changed, wearerFid: wearerFidChanged } = dirtyFields;
     
         try {
+            // TODO: need to we aware of which FID is currently selected
+            const fid = fidItems[1].id as number;
```

Comment:
Need to make the selected FID available here somehow, so that we know which user's signing keys to use when creating new msgs